docs/Release Notes/12.3.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Version 12.3
+parent: Release Notes
+permalink: /docs/release-notes/2022/12.3/
+nav_order: 90
+---
+
+# 12.3
+
+{: .no_toc }
+
+---
+
+### New Functionalities
+
+* iOS - Now it's possible to push the SDK into your own UINavigationController instead of presenting by modal:
+    * <a href="https://cartrawler.github.io/docs/ios/standalone/implementation-steps#present-the-sdk-via-push-presentation" target="_blank">https://cartrawler.github.io/docs/ios/standalone/implementation-steps#present-the-sdk-via-push-presentation</a>
+
+### Sizing guidelines
+The details below should be used as a guideline around SDK size related to release.
```

Comment:
The details below provide guidance related to the footprint of the CarTrawler SDK.
Maybe call the heading 'CarTrawler SDK Footprint'?